importer/api/__init__.py

```diff
@@ -148,7 +148,7 @@ def import_children(course, element, parent, parent_dpath):
         None
     """
     # pylint: disable=too-many-locals
-    title = element.attrib.get("display_name", "MISSING")
+    title = element.attrib.get("display_name", "...")
```

Comment:
I do not understand why 
https://github.com/mitodl/lore/blob/fix/aq/fix_display_name_lr/search/search_indexes.py#L121
gives a problem.
In theory having `from __future__ import unicode_literals` at the beginning means all the strings are in unicode.
Can you make sure that all the modules where `title` is used make that import?

did you try to use the string `"…"` directly instead of `"MISSING"` or did you use the unicode code?